client/components/FoodDetails.jsx

```diff
@@ -1,38 +1,64 @@
 import React from 'react'
```

Comment:
This file seems unrelated to this ticket.

Reply:
? i think this line needs to be there